### DISCUSSION TITLE
Mortgage Caculator Textbox design!

### DISCUSSION THREAD

**Discussion by GEONHWISONG** [PranishKansakar/Tafesa240154#12] [General]
Hey guys! I'm happy to hear your opinions about the design of the textbox in the Mortgage Calculator. I would like to create a rectangular shape with a bit of curve on all edges. Please share your opinions! Thanks!

**Comment by PranishKansakar** [comment]
Yes, creating a rectangular shape with slightly curved edges for the textbox in the Mortgage Calculator sounds like a visually appealing and modern design choice.
